tests/parm/control_progsigma.nml.IN

```diff
@@ -0,0 +1,354 @@
+  do_sat_adj = .true.
+  consv_am = .false.
+  fill = .true.
+  dwind_2d = .false.
+  print_freq = 6
+  warm_start = @[WARM_START]
+  no_dycore = .false.
+  z_tracer = .true.
+  agrid_vel_rst = .true.
+  read_increment = @[READ_INCREMENT]
+  res_latlon_dynamics = @[RES_LATLON_DYNAMICS]
+/
+
+&external_ic_nml
+  filtered_terrain = .true.
+  levp = @[NPZP]
+  gfs_dwinds = .true.
+  checker_tr = .false.
+  nt_checker = 0
+/
+
+&gfs_physics_nml
+  fhzero       = @[FHZERO]
+  h2o_phys     = .true.
+  ldiag3d      = @[LDIAG3D]
+  qdiag3d      = @[QDIAG3D]
+  print_diff_pgr = @[PRINT_DIFF_PGR]
+  fhcyc        = @[FHCYC]
+  use_ufo      = .true.
+  pre_rad      = .false.
+  imp_physics  = 11
+  pdfcld       = .false.
+  fhswr        = 3600.
+  fhlwr        = 3600.
+  ialb         = 1
+  iems         = 1
+  iaer         = @[IAER]
+  icliq_sw     = 2
+  iovr         = @[IOVR]
+  ico2         = 2
+  isubc_sw     = 2
+  isubc_lw     = 2
+  isol         = 2
+  lwhtr        = .true.
+  swhtr        = .true.
+  cnvgwd       = .true.
+  shal_cnv     = .true.
+  cal_pre      = .false.
+  redrag       = .true.
+  dspheat      = .true.
+  hybedmf      = .false.
+  satmedmf     = .true.
+  isatmedmf    = 1
+  lheatstrg    = @[LHEATSTRG]
+  lseaspray    = @[LSEASPRAY]
+  random_clds  = @[RANDOM_CLDS]
+  trans_trac   = .true.
+  cnvcld       = @[CNVCLD]
+  imfshalcnv   = @[IMFSHALCNV]
+  imfdeepcnv   = @[IMFDEEPCNV]
+  ras          = @[RAS]
+  cdmbgwd      = @[CDMBWD]
+  prslrd0      = 0.
+  ivegsrc      = 1
+  isot         = 1
+  lsoil        = 4
+  lsm          = 1
+  iopt_dveg    = 1
+  iopt_crs     = 1
+  iopt_btr     = 1
+  iopt_run     = 1
+  iopt_sfc     = 1
+  iopt_frz     = 1
+  iopt_inf     = 1
+  iopt_rad     = 1
+  iopt_alb     = 2
+  iopt_snf     = 4
+  iopt_tbot    = 2
+  iopt_stc     = 1
+  iopt_trs     = 2
+  debug        = .false.
+  oz_phys      = .false.
+  oz_phys_2015 = .true.
+  nstf_name    = 2,0,0,0,0
+  nst_anl      = .true.
+  psautco      = 0.0008,0.0005
+  prautco      = 0.00015,0.00015
+  lgfdlmprad   = .true.
+  cplchm       = @[CPLCHM]
+  cplwav       = @[CPLWAV]
+  cplwav2atm   = @[CPLWAV2ATM]
+  effr_in      = .true.
+  ldiag_ugwp   = .false.
+  do_ugwp      = .false.
+  do_tofd      = .true.
+  do_sppt      = @[DO_SPPT]
+  do_shum      = @[DO_SHUM]
+  do_skeb      = @[DO_SKEB]
+  lndp_type    = @[LNDP_TYPE]
+  n_var_lndp   = @[N_VAR_LNDP]
+  fscav_aero   = @[FSCAV_AERO]
+  do_RRTMGP          = @[DO_RRTMGP]
+  active_gases       = 'h2o_co2_o3_n2o_ch4_o2'
+  ngases             = 6
+  lw_file_gas        = 'rrtmgp-data-lw-g128-210809.nc'
+  lw_file_clouds     = 'rrtmgp-cloud-optics-coeffs-lw.nc'
+  sw_file_gas        = 'rrtmgp-data-sw-g112-210809.nc'
+  sw_file_clouds     = 'rrtmgp-cloud-optics-coeffs-sw.nc'
+  rrtmgp_nGptsSW     = 112
+  rrtmgp_nGptsLW     = 128
+  rrtmgp_nBandsLW    = 16
+  rrtmgp_nBandsSW    = 14
+  doGP_cldoptics_LUT = @[DOGP_CLDOPTICS_LUT]
+  doGP_lwscat        = @[DOGP_LWSCAT]
+  progsigma          = .true.     
```

Comment:
The file control_prosigma.nml.IN looks identical to control.nml.IN with the addition of the prosigma variable. Please instead add a new variable in default_vars (export PROSIGMA=.false.) and then set it true in your control_c384_prosigma test. You would not need to add the control_prosigma.nml.IN.

Reply:
Ok, sure I can do that.

Reply:
@lisa-bengtsson so is the PR ready to run regression test? I mean where to use PROSIGMA option in control_prosigma or control_c384_prosigma?

Reply:
@lisa-bengtsson sure!